terraform/sa.tf

```diff
@@ -49,6 +49,18 @@ data "aws_iam_policy_document" "service_account" {
       "arn:aws:lambda:${var.region}:${var.target_aws_account_id}:function:ien*",
     ]
   }
+
+  statement {
```

Comment:
can you add a comment of the final IAM, I can send it over to get the SA updated.

Reply:
```{
  "public_url": {
    "sensitive": false,
    "type": "string",
    "value": "d309kopm8ags5f.cloudfront.net"
  },
  "service_account_iam": {
    "sensitive": false,
    "type": "string",
    "value": {
      "Version": "2012-10-17",
      "Statement": [
        {
          "Sid": "AllowS3FullAccess",
          "Effect": "Allow",
          "Action": [
            "s3:PutObjectAcl",
            "s3:PutObject",
            "s3:ListBucket",
            "s3:GetObject",
            "s3:GetBucketLocation",
            "s3:DeleteObject"
          ],
          "Resource": "arn:aws:s3:::ien-*"
        },
        {
          "Sid": "AllowCFInvalidationAccess",
          "Effect": "Allow",
          "Action": [
            "cloudfront:UpdateDistribution",
            "cloudfront:UpdateCloudFrontOriginAccessIdentity",
            "cloudfront:TagResource",
            "cloudfront:ListInvalidations",
            "cloudfront:ListDistributions",
            "cloudfront:GetInvalidation",
            "cloudfront:GetDistributionConfig",
            "cloudfront:GetDistribution",
            "cloudfront:CreateInvalidation",
            "cloudfront:CreateDistributionWithTags",
            "cloudfront:CreateCloudFrontOriginAccessIdentity",
            "cloudfront:CreateCachePolicy"
          ],
          "Resource": "arn:aws:cloudfront::588188670678:distribution/*"
        },
        {
          "Sid": "AllowLambdaInvokeDeploy",
          "Effect": "Allow",
          "Action": [
            "lambda:UpdateFunctionCode",
            "lambda:InvokeFunction"
          ],
          "Resource": "arn:aws:lambda:ca-central-1:588188670678:function:ien*"
        },
        {
          "Sid": "AllowDBClusterSnapshot",
          "Effect": "Allow",
          "Action": [
            "rds:AddTagsToResource",
            "rds:CreateDBClusterSnapshot"
          ],
          "Resource": "arn:aws:lambda:ca-central-1:588188670678:cluster:ien*"
        }
      ]
    }
  }
}```

Reply:
Sent to the platform team